mflowgen/glb_top/custom-init/outputs/floorplan.tcl

```diff
@@ -21,21 +21,23 @@ set core_margin_r [expr 5 * $horiz_pitch]
 set core_margin_l [expr 5 * $horiz_pitch]
 
 # Margins between glb tiles and core edge
-set tile_margin_t [expr 5 * $vert_pitch]
-set tile_margin_b [expr 120 * $vert_pitch]
-set tile_margin_l [expr 150 * $horiz_pitch]
-set tile_margin_r [expr 150 * $horiz_pitch]
+set tile_margin_t [expr 100 * $vert_pitch]
+set tile_margin_b [expr 50 * $vert_pitch]
+set tile_margin_l [expr 500 * $horiz_pitch]
```

Comment:
Is the idea here to have the non-sram logic on the left of the GLB tiles rather than below?

Reply:
Yes. Some clock gating generating logic and registers should be located at the left of the first `glb-tile`. So I increased the margin at the left. The exact value may change in with gf.